src/HydraAuction/Tx/Deposit.hs

```diff
@@ -161,7 +163,7 @@ losingBidderClaimDeposit terms = do
         , outs = []
         , toMint = TxMintValueNone
         , changeAddress = bidderAddress
-        , validityBound = (Just $ biddingEnd terms, Nothing)
+        , validityBound = stageToInterval terms BiddingEndedStage
```

Comment:
This is incorrect.

Left logic has no upper bound, while stage has.

Reply:
Solution I had in mind was intervals over stage, maybe I formulate that in issue description badly.

While seems like stage-to-posinf is the only case we do use.

Reply:
You are right. Maybe we can just add stage-to-posinf in utils as well, and use that?

Reply:
> we can just add stage-to-posinf in utils as well

But that is domain (stage) specific thing, no?